docs/release-notes/RELEASE-NOTES-1.3.0.md

```diff
@@ -38,9 +55,9 @@ The node user API is documented:
 * WebSocket API endpoints are [specified online][api-doc];
 * The intended usage of the user API (HTTP and WebSocket) is [documented online][api-doc].
 
-[swagger-yaml]: https://github.com/aeternity/epoch/blob/v1.3.0/config/swagger.yaml
-[swagger-ui]: https://aeternity.github.io/epoch-api-docs/?config=https://raw.githubusercontent.com/aeternity/epoch/v1.3.0/apps/aehttp/priv/swagger.json
-[api-doc]: https://github.com/aeternity/protocol/blob/epoch-v1.3.0/epoch/api/README.md
+[swagger-yaml]: https://github.com/aeternity/aeternity/blob/v1.3.0/config/swagger.yaml
+[swagger-ui]: https://aeternity.github.io/api-docs/?config=https://raw.githubusercontent.com/aeternity/aeternity/v1.3.0/apps/aehttp/priv/swagger.json
+[api-doc]: https://github.com/aeternity/protocol/blob/aeternity-node-v1.3.0/node/api/README.md
```

Comment:
Partially delivers https://www.pivotaltracker.com/story/show/163321210

Update on the protocol repo side to be done yet. Not a blocker for this PR.